core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -101,7 +101,9 @@ describe('Enable correctness feedback and set correctness', function() {
     // Create interaction first.
     await explorationEditorMainTab.setInteraction('MultipleChoiceInput', [
       await forms.toRichText('Correct!'),
-      await forms.toRichText('Wrong!')
+      await forms.toRichText('Wrong!'),
+      await forms.toRichText('Not correct'),
+      await forms.toRichText('Wrong again')
     ]);
     await explorationEditorMainTab.addResponse(
       'MultipleChoiceInput', await forms.toRichText('Good!'),
```

Comment:
Hey @EricZLou 
 just wanted to ask that here we are doing `await forms.toRichText('Good!')` but we don't have an option for that and we are doing it equal to the `correct!`. Will replacing one of the options with `Good!` work?
Thanks

Reply:
Feel free to change it! You can see the parameters to `addResponse` in `ExplorationEditorMainTab.js`. The 2nd parameter is `feedbackInstructions`, so it represents the action that occurs when the lesson gives feedback to the user after an interaction input. So it can be something like `await forms.toRichText('This is a default response')` and it can also be `null`.

Reply:
Yeah, that makes sense, it won't affect as it's just the feedback.
Thanks for clearing it up @EricZLou.